index.Rmd

```diff
@@ -31,37 +31,37 @@ description: "A handbook on teaching and learning with Jupyter notebooks."
 # Introduction {#intro}
 
 This handbook is for any educator teaching a topic that includes data analysis
-or computation to support the learning—not just courses in engineering or
+or computation in order to support learning. It is not just for educators teaching courses in engineering or
 science, but also data journalism, business and quantitative economics,
 data-based decision sciences and policy, quantitative health sciences, and
-digital humanities. It aims to give an entry point, and a broad overview of
+digital humanities. It aims to provide an entry point, and a broad overview of
 Jupyter in education. Whether you are already using Jupyter to teach, you have
 found learning materials built on Jupyter that piqued your curiosity, or have
-never heard of Jupyter, the material in this open book can help you empower your
-teaching with this new technology.
+never heard of Jupyter, the material in this open book can empower you to 
+use this technology in your teaching. 
 
 [Project Jupyter](http://jupyter.org/) is a broad collaboration that develops
 open-source tools for interactive and exploratory computing. The tools include:
 over 100 computer languages (with a focus on Python), the Jupyter Notebook, JupyterHub,
 and an ecosystem of extensions
-contributed by a large community. Jupyter Notebook exploded in popularity since
+contributed by a large community. The Jupyter Notebook has exploded in popularity since
 late 2014, fueled by its adoption as the favorite environment for doing data
-science. It has also grown as a platform to use in the classroom, to develop
-teaching materials, to share lessons and tutorials, and to create
-computational stories. Notebooks are documents containing text narratives,
+science. It has also grown as a platform to use in the classroom, to develop 
+teaching materials, to the share lessons and tutorials, and to the create
```

Comment:
@lheagy Were the `the`s a copy/pasta leftover?

Reply:
thanks @willingc, yes - on it!